### PR TITLE
Support single expression members

### LINKED ISSUE #72: Support omitting 'return' keyword
**Is your feature request related to a problem? Please describe.**

[Swift 5.1 introduced the ability](https://www.swiftbysundell.com/tips/omitting-the-return-keyword/) to omit the `return` keyword, if functions only contain a single expression.
This is currently not translated and leads to a missing `return` keyword in Kotlin.

**Describe the solution you'd like**

Ideally, if a function has only one expression that returns a value (so not `Void`), it should be preceded with a `return` if there is none.

**Describe alternatives you've considered**

Forcing a Swift user to add `return` statements where they are not necessary would be the alternative, and current, solution. This works but decreases the convenience in using Gryphon.

**Additional context**

Something like this:

```Swift
func example() -> String {
    "Hello, World"
}
```

should be translated to:

```Kotlin
internal fun example(): String {
   return "Hello, World"
}
```

(in Gryphon 0.9.0, the `return` keyword is missing)

### PR DESCRIPTION
**Pull request by sgade**
### What's in this pull request?

Adds support for single-expression members (on classes and structs).
This is part 2, following #75.

Example:

```swift
class SingleExpressionMembers {
	var one: Int {
		10
	}

	var two: Int {
		get {
			20
		}
	}
}
```

```kotlin
internal open class SingleExpressionMembers {
	open val one: Int
		get() = 10
	open val two: Int
		get() = 20
}

```

### Does this resolve an open issue?
Yeah, it resolves #72.

### Checklist for submitting a pull request:

- [x] Your code builds without any errors or warnings (warnings when running `prepareForBootstrapTests.sh` are OK).
- [x] You ran the unit tests and Bootstrapping tests for your platform.
  - [x] If you changed any `.kt` files in the `Test cases` folder, you also ran the Acceptance tests.
  
- [x] You have added new tests that check your changes (if possible).
- [x] This pull request is targeting the `development` branch (if you're contributing code) or the `gh-pages` branch (if you're contributing to the website).

### CODE REVIEW

File: Sources/GryphonLib/KotlinTranslator.swift [hunk 1]
```diff
@@ -1262,12 +1262,21 @@ public class KotlinTranslator {
...
-					statements,
-					withIndentation: indentation2,
-					limitForAddingNewlines: 3))
-				result.append(indentation1 + "}\n")
+				result.append("\(indentation1)get() ")
+
+				if let singleExpressionStatement = isSingleExpressionFunction(statements, returnTypeString: getter.returnType) {
+					result.append("= ")
+					result.append(try translateExpression(singleExpressionStatement.expression,
+														  withIndentation: indentation1))
```
Comment (vinivendra) [inline Sources/GryphonLib/KotlinTranslator.swift:1270]: Could you add a test that uses this indentation? Maybe a long function like before, or a closure with a statement in it if you prefer.

**Comment by vinivendra** [comment]
Ok, the release is ready, just waiting for this. Let me know if you'll have time to add the test now, otherwise I'll merge the PR and we can add it later.

**Comment by vinivendra** [comment]
Alright, I'm merging it (sorry, I'm kind of in a hurry here).
Great job and thanks again :)

File: Sources/GryphonLib/KotlinTranslator.swift (hunk 1 above)
Comment (sgade) [reply Sources/GryphonLib/KotlinTranslator.swift:1270]: I added it in #82. Needed to get some sleep last night. 😉

**Pull request merged**